client/lib/paths/login/index.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * Internal dependencies
+ */
+import { addQueryArgs } from 'lib/url';
+import config, { isEnabled } from 'config';
+
+export function login( { redirectTo } ) {
+	const url = isEnabled( 'wp-login' ) ? '/login' : config( 'login_url' );
```

Comment:
I think we can hardcode the path for desktop [here](https://github.com/Automattic/wp-calypso/blob/master/client/auth/controller.js#L45) actually. I'm not sure how much sense it makes to have it in the config just for desktop since we have a special case for it anyway